nanome/_internal/_structure/_helpers/_conformer_helper.py

```diff
@@ -189,7 +188,7 @@ def convert_to_conformers(complex, force_conformer = None): #Data.Complex -> Dat
             bond_total_count+=1
 
         for atom in molecule.atoms:
-            for bond in molecule.bonds:
+            for bond in atom.bonds:
```

Comment:
This was where the problem was. This typo led to 5000 iterations being run instead of 50. exponents amirite